frontend/components/PhoneinputTest.tsx

```diff
@@ -0,0 +1,142 @@
+import React, { Component, useRef } from "react";
+import {
+  SafeAreaView,
+  StyleSheet,
+  View,
+  StatusBar,
+  TouchableOpacity,
+  Text,
+  Pressable,
+} from 'react-native';
+import PhoneInput from 'react-native-phone-number-input';
+// import {Colors} from 'react-native/Libraries/NewAppScreen';
+// const phoneInput = useRef<PhoneInput>(null);
+type State = {
+    value: string,
+    countryCode: string,
+    formattedValue: string,
+    valid:boolean,
+    disabled: boolean,
+    showMessage: boolean,
+
+}
+class PhoneinputTest extends Component<{}, State> {
+    phoneInput=React.createRef<PhoneInput>();
+    constructor(props: {}) {
+      super(props);
+      this.state = {
```

Comment:
Minor: If constructor is not doing anything major other than setting state, better use class variable `state = {...}` reduces lines of code and improved readability